_drgn.pyi

```diff
@@ -959,6 +959,9 @@ class Architecture(enum.Enum):
     RISCV32 = ...
     """The 32-bit RISC-V architecture."""
 
+    S390 = ...
+    """The s390 architecture, a.k.a. IBM Z."""
```

Comment:
I've seen the names s390 and s390x. Would you happen to know what the difference is?

Reply:
s390 is the older architecture name when it was 31 bit. Somewhere around 2000 it was extended to 64 bit, and than it was called 's390x'. The kernel however kept the 's390' name as architecture, while userspace programs usually use s390x. As drgn is a kernel debugger, i would name it the same way as the architecture in the kernel.